client/landing/stepper/declarative-flow/internals/steps-repository/playground/lib/blueprint.ts

```diff
@@ -105,14 +105,35 @@ function getDefaultBlueprint( recommendedPhpVersion: string ): Blueprint {
 	};
 }
 
-export function getBlueprintName( name: string | null ): string | null {
+function getBlueprintName( name: string | null ): string | null {
 	if ( name && name in PREDEFINED_BLUEPRINTS ) {
 		return name;
 	}
 
 	return null;
 }
 
+// Used in sending the Tracks event
+export function getBlueprintLabelForTracking( query: URLSearchParams ): string {
+	const name = query.get( 'blueprint' );
+
+	if ( name && name in PREDEFINED_BLUEPRINTS ) {
+		return name;
+	}
+
+	// If its a blueprintlibrary.wordpress.com url for blueprint, use its id to construct the label
```

Comment:
Little typo here: "If **it's** a blueprintlibrary.wordpress.com URL for blueprint, use its id to construct the label"

Reply:
Done!